CODEOWNERS

```diff
@@ -7,6 +7,8 @@
 # These are the default owners for the whole content of the `kyma` repository. The default owners are automatically added as reviewers when you open a pull request, unless different owners are specified in the file.
 * @derberg @pbochynski @PK85 @a-thaler @piotrmsc
 
+/scripts/ @mszostok @aszecowka @piotrmiskiewicz @sjanota @Tomasz-Smelcerz-SAP @strekm @crabtree @akgalwas @janmedrek @Szymongib @sayanh @radufa @suleymanakbas91
```

Comment:
do you have the consent of all the people listed here that they can maintain those scripts?
btw, please describe new dir here https://github.com/kyma-project/kyma#project-structure

Reply:
I copied the list from the `common` folder. I'll add it to the structure.

Reply:
We need there people that will actually do reviews and feel responsible for the content of the directory. So if you do not want to ask there one by one people maybe just for now put yourself and maybe @michal-hudy that owned the initial proposal?

Reply:
👍